planning/behavior_velocity_planner/src/scene_module/virtual_traffic_light/scene.cpp

```diff
@@ -493,8 +493,7 @@ bool VirtualTrafficLightModule::isAfterAnyEndLine()
   const auto signed_arc_length = tier4_autoware_utils::calcSignedArcLength(
     module_data_.path.points, module_data_.head_pose.position, collision->point);
 
-  constexpr double max_excess_distance = 3.0;
-  return signed_arc_length < -max_excess_distance;
+  return signed_arc_length < -planner_param_.near_line_distance;
```

Comment:
@.kosuke55 
I think this would be better to be dead_line_margin like other module

Reply:
@taikitanaka3 
I changed params of `isBeforeStopLine()` and `isAfterAnyEndLine()` to `dead_line_margin`. https://github.com/autowarefoundation/autoware.universe/pull/228/commits/6c64909ee6fddc9f7b46461f9153681817e04467